games-arcade/caveexpress/caveexpress-2.4.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Classic 2D platformer with physics-based gameplay and dozens of levels"
+DESCRIPTION="Master your pedal-powered flying machine to pick up packages \
+from your cave-dwelling clients and drop them off at the collection point. \
+But beware! Mighty mastodons, terrifying pterodactyls and others \
+would rather see you extinct."
```

Comment:
The description needs to describe the included cavepacker game also, or we can just disable cavepacker via `-DCAVEPACKER=OFF` to cmake.

It's actually better to disable cavepacker and package it separately, as they use different versioning scheme.

Reply:
@alviss Maybe let's create a issue for this recipe and pack it together for now?